peer.go

```diff
@@ -583,16 +86,16 @@ func (p *peer) pushBlockMsg(sha *wire.ShaHash, doneChan, waitChan chan struct{})
 // the connected peer.  Since a merkle block requires the peer to have a filter
 // loaded, this call will simply be ignored if there is no filter loaded.  An
 // error is returned if the block hash is not known.
-func (p *peer) pushMerkleBlockMsg(sha *wire.ShaHash, doneChan, waitChan chan struct{}) error {
+func pushMerkleBlockMsg(p *peer.Peer, s *server, sha *wire.ShaHash, doneChan, waitChan chan struct{}) error {
```

Comment:
Same deal here as with `pushTxMsg` regarding moving to `server.go` and defining it before `handleGetData`.